### ISSUE TITLE
Is anyone updating the melpa version of anki-editor?

### ISSUE THREAD

**Issue by falematte** [orgtre/anki-editor#25]

**Comment by orgtre** [comment]
No. The source of the MELPA version is https://github.com/louietan/anki-editor and the code there has not been updated for more than three years.

We have started the process of trying to link this repo to MELPA instead [here](https://github.com/orgtre/anki-editor/issues/20). I've tried to contact the original author and once enough time has passed to allow for a response, we'll move ahead.
